l1-contracts/contracts/bridge/AssetRouterBase.sol

```diff
@@ -0,0 +1,179 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.24;
+
+import {Ownable2StepUpgradeable} from "@openzeppelin/contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
+import {PausableUpgradeable} from "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import {IAssetRouterBase} from "./interfaces/IAssetRouterBase.sol";
+import {IL2Bridge} from "./interfaces/IL2Bridge.sol";
```

Comment:
Very good, we're starting to get there. This file should not import IL2Bridge, but BridgeBase, which can be inherited by IL2Bridge and IL1Bridge. 

Actually it might not even be used. But the point remains, to remove L1 /L2 distinctions

Reply:
+